src/App.js

```diff
@@ -25,6 +26,8 @@ function App() {
     const storedbingo = JSON.parse(localStorage.getItem(LOCAL_STORAGE_KEY))
     if (storedbingo) {
       return (storedbingo)
+    } else {
```

Comment:
This is the update not to have an empty card and missing button when local storage is missing